checklists/appsvc_checklist.json

```diff
@@ -0,0 +1,304 @@
+      "guid": "cda3b54e-b2eb-403d-b9a2-582718d2ddb1",
+      "severity": "Low",
+      "link": "https://docs.microsoft.com/azure/app-service/networking/nat-gateway-integration"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
+      "text": "Inbound network access should be controlled",
+      "description": "Control inbound network access using a combination of App Service Access Restrictions, Service Endpoints or Private Endpoints.",
+      "guid": "0725769e-e669-41a4-a34a-c932223ece80",
+      "severity": "High",
+      "link": "https://docs.microsoft.com/azure/app-service/networking-features#access-restrictions"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
+      "text": "Use a WAF in front of App Service",
+      "description": "Protect against malicious inbound traffic using a Web Application Firewall like Application Gateway or Azure Front Door.  Make sure to monitor the WAF's logs.",
+      "guid": "b123071a-5416-4415-a33e-a3ad2c2de732",
+      "severity": "High",
+      "link": "https://docs.microsoft.com/azure/app-service/networking/app-gateway-with-service-endpoints"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
+      "text": "Avoid for WAF to be bypassed",
+      "description": "Make sure the WAF cannot be bypassed by locking down access to only the WAF.  Use a combination of Access Restrictions, Service Endpoints and Private Endpoints.",
+      "guid": "165c3acb-ef4a-4be1-b8d3-9fda47768314",
+      "severity": "High",
+      "link": "https://docs.microsoft.com/azure/app-service/networking-features#access-restrictions"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
+      "text": "Set minimum TLS policy to 1.2",
+      "description": "Set minimum TLS policy to 1.2 in App Service configuration.",
+      "guid": "c115775c-2ea5-45b4-9ad4-8408ee72734b",
+      "severity": "Medium",
+      "link": "https://docs.microsoft.com/azure/app-service/configure-ssl-bindings#enforce-tls-versions"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
+      "text": "Use HTTPS only",
+      "description": "Configure App Service to use HTTPS only.  This causes App Service to redirect from HTTP to HTTPS.  Strongly consider the use of HSTS in your code or from your WAF.",
+      "guid": "475ba18f-dbf5-490c-b65d-e8e03f9bcbd4",
+      "severity": "High",
+      "link": "https://docs.microsoft.com/azure/app-service/configure-ssl-bindings#enforce-https"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
```

Comment:
Not really network security this.

Reply:
Why not? CORS is restricting the browser to make network requests unless explicitly allowed?